webapp/src/main/webapp/themes/wilma/css/wilma.css

```diff
@@ -76,6 +76,207 @@ h5 {
     font-size: .875em;
     color: #064d68;
 }
+
+.clearfix::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+
+/* ------------------------------------------------->  */
+/* BOOTSTRAP MODIFICATION -------------------------->  */
+/* ------------------------------------------------->  */
+
+
+.container,
+.container-fluid,
+.container-xxl,
+.container-xl,
+.container-lg,
+.container-md,
+.container-sm {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+    --bs-gutter-x: 1.5rem;
+    --bs-gutter-y: 0;
+    display: flex;
+    flex-wrap: wrap;
+    /* margin-top: calc(-1 * var(--bs-gutter-y));
+    margin-right: calc(-0.5 * var(--bs-gutter-x));
+    margin-left: calc(-0.5 * var(--bs-gutter-x)); */
+  }
+  .row > * {
+    box-sizing: border-box;
+    flex-shrink: 0;
+    width: 100%;
+    max-width: 100%;
+    /* padding-right: calc(var(--bs-gutter-x) * 0.5);
+    padding-left: calc(var(--bs-gutter-x) * 0.5); */
+    /* margin-top: var(--bs-gutter-y); */
+  }
+
+.col-1 {
```

Comment:
I found only col-md5 used once and col-md7 used once. All other 'col-' classes aren't used, right?
The same is related to container-xxl, container-fluid, container-xl, container-lg, container-md, container-sm
If that is the case I would suggest to clean up not used classes.